_security/access-control/permissions.md

```diff
@@ -65,6 +65,47 @@ Rather than individual permissions, you can often achieve your desired security
 {: .tip }
 
 
+## System index permissions
+
+System index permissions are unique among other permissions in that they extend some traditional admin-only accessibility to non-admin users. These permissions give normal users the ability to modify any system index specified in the role or roles to which they are mapped. The exception to this is the security system index, `.opendistro_security`, which is used to store Security's configuration YAML files and remains accessible only to admins with an admin certificate.
+
+Along with standard index permissions, you specify system index permissions in the 'roles.yml' configuration file under 'index_permissions' (See [roles.yml]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#rolesyml). You do this by specifying `system:admin/system_index` in the role's `allowed_actions` section and adding the system index in the `index_patterns` section.
+
+For example, the system index permission that gives a user permission to modify the system index that stores configurations for the Alerting plugin is defined by the index pattern `.opendistro-alerting-config` and its allowed action is defined as `system:admin/system_index`. The following role shows how this system index permission is configured along with other attributes:
+
+```yml
+alerting-role:
+  reserved: true
+  hidden: false
+  cluster_permissions:
+  - "cluster:admin/opendistro/alerting/alerts/ack"
+  - "cluster:admin/opendistro/alerting/alerts/get"
+  index_permissions:
+  - index_patterns:
+    - ".opendistro-alerting-config"
+  - allowed_actions:
+    - "system:admin/system_index"
+```
+
+The permission prefix `.opendistro` also works with the wildcard to extend its reach of access. This can be useful, but it should be used with caution to avoid giving unintentional access to system indexes. When specifying system indexes for roles, keep the following considerations in mind:
+
+* Specifying the full name of a system index limits access to that index alone: `.opendistro-alerting-config`.
+* Specifying the prefix and a partial name for a system index provides access to all system indexes that begin with the name: `.opendistro-anomaly-detector*`.
+* Using `.*` is effectively the same as specifying the prefix with wildcard, as described in the previous point. This gives access to all system indexes that begin with a `.`.
```

Comment:
Using `*` is allowed, there is no requirement for a special case with `.*`. The following index pattern is the shortest way to give a user full access to all - including system indices - on the cluster.

```
  index_permissions:
    - index_patterns:
        - '*'
      allowed_actions:
        - '*'
        - 'system:admin/system_index'
```

Reply:
[Quotes Single vs Double]
Just looked this up from from the [spec](https://yaml.org/spec/1.2.2/#731-double-quoted-style).  Generally we shouldn't use quotes, however, special characters like `*` and `:` should be quoted.

Generally single quote style is what we should use in our docs as double quotes are used for interpolation eg `\n` becomes the newline character inside double quotes.

Don't want to think about it, you can interactively use [yaml-formatter](https://jsonformatter.org/yaml-formatter), or we can look at added a formatter as part of this project.

Reply:
> Is the additional wildcard necessary under allowed_actions only when you want to specify all system indexes under index_patterns?

@DarshitChanpura can you confirm?

Reply:
@peternied "YAML formatter": a thing of beauty. Thanks.